test/functionalTest/cases/1367_time_measures/time_measure_stats.test

```diff
@@ -31,7 +31,7 @@ brokerStart CB 101,38 IPv4 -reqTimeStat
 --SHELL--
 
 #
-# 01. Initial call to GET /statistics (gives NO time stats as LAST /statistics is excluded)
+# 01. Pre-call to GET /statistics to give VALGRIND a possibility
```

Comment:
Either that or sending that "hidden" request.
Or even better, make valgrindTestSuite and testHarness.sh behave in the exact same way

I picked the fastest option which was to send a hidden request ...

Fixed in 28ee59a

If you think this is not enough, I'd propose to create an issue about it.